lib/oga/xml/parser.rll

```diff
@@ -245,7 +245,7 @@ string_body
   def reset
     @line = 1
 
-    @lexer.reset
+    @lexer.reset_native
   end
```

Comment:
This `reset` method can also just be removed as parser instances aren't re-used.